.openshift-ci/jobs/update-support-packages/run.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+set -eoux pipefail
+
+gcloud_command() {
+    gcloud compute ssh --ssh-key-file="$GCP_SSH_KEY_FILE" "$GCLOUD_USER"@"$GCLOUD_INSTANCE" -- -T "$1"
+}
+
+sleep_echo() {
+    for _ in {1..120}; do
+        sleep 1
+        echo -n .
+    done
+    echo
+}
+
+export WORKDIR=/go/src/github.com/stackrox/collector
+
+source "$WORKDIR/.openshift-ci/drivers/scripts/lib.sh"
+
+BRANCH="$(get_branch)"
+
+# shellcheck source=SCRIPTDIR/../../drivers/scripts/lib.sh
+source "${WORKDIR}/.openshift-ci/drivers/scripts/lib.sh"
+
+if ! pr_has_label "test-support-packages"; then
+    if [ "${BRANCH}" != "master" ]; then
+        exit 0
+    fi
+fi
+
+# shellcheck source=SCRIPTDIR/../../jobs/update-support-packages/env.sh
+source "${WORKDIR}/.openshift-ci/jobs/update-support-packages/env.sh"
+
+env
+
+gcloud_command "git clone https://github.com/stackrox/collector.git --single-branch --branch=$BRANCH --depth=1"
```

Comment:
Can this and the following line be also put into the run-all.sh?

Reply:
I would like it to be possible to run run-all.sh locally. That is why I would not like to clone the repo or install zip in run-all.sh.